docs/content/author-apps/kubernetes-resources/snippets/kubernetes-resource.bicep

```
@@ -0,0 +1,23 @@
+import kubernetes as kubernetes {
+  kubeConfig: '****'
```

```suggestion
  kubeConfig: ''
```

We should leave this blank instead of putting in garbage text